Examples/KlaviyoSwiftExamples/Shared/AppDelegate.swift

```diff
@@ -34,14 +34,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.makeKeyAndVisible()
 
         // STEP2: Setup Klaviyo SDK with api key
-        Klaviyo.setupWithPublicAPIKey(apiKey: "magpcN")
+        KlaviyoSDK().initialize(with: "magpcN")
 
         // EXAMPLE: of how to track an event
-        Klaviyo.sharedInstance.trackEvent(eventName: "Opened kLM App")
+        KlaviyoSDK().create(event: .init(name: .CustomEvent("Opened kLM App")))
```

Comment:
You could keep an instance around if you want too.

Reply:
What is the best practice we want to suggest to our users? Keep an instance or use `KlaviyoSDK()` whenever they want to access the SDK.